src/main/java/com/scalar/db/api/PutIf.java

```diff
@@ -26,7 +26,18 @@
   public PutIf(ConditionalExpression... expressions) {
     checkNotNull(expressions);
     this.expressions = new ArrayList<>(expressions.length);
-    Arrays.stream(expressions).forEach(v -> this.expressions.add(v));
+    this.expressions.addAll(Arrays.asList(expressions));
```

Comment:
This change is something we can do in a small PR and merge it before this?

Reply:
I have created a PR for it:
https://github.com/scalar-labs/scalardb/pull/211

Please take a look! After it merged, I will push a commit to revert those changes.